ranking/opts.go

```diff
@@ -57,10 +57,14 @@ func boolFromQuery(query url.Values, name string) bool {
 }
 
 func addSuffixesForFiletype(suffixes *map[string]float32, filetype string) {
+	// what exactly happens here? what does the number assign do?
+	// i'm just trying to add support for Objective-C source files
 	switch filetype {
 	case "c":
 		(*suffixes)[".c"] = 0.75
 		(*suffixes)[".h"] = 0.75
+	case "m":
+		(*suffixes)[".m"] = 0.75
```

Comment:
No .m files are together with .h files. Feel free to have a look at http://packages.debian.org/volumecontrol.app for an example.